src/style.css

```diff
@@ -0,0 +1,128 @@
+.page-html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+
+  color: #060b35;
+}
+
+.reset {
+  margin: 0;
+}
+
+.product-card {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-content: flex-start;
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+}
+
+.product-card__img {
+  height: 134px;
+  width: 160px;
+}
+
+.product-card__name {
+  padding-top: 40px;
+
+  font-weight: 500;
+  line-height: 18px;
+  font-size: 12px;
+}
+
+.product-card__code {
+  flex-basis: 100%;
+  padding-top: 4px;
+  width: 98px;
+
+  line-height: 14px;
+  font-size: 10px;
+
+  color: #616070;
+}
+
+.star-box {
```

Comment:
[Checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md) the 11th point